.pre-commit-config.yaml

```diff
@@ -33,7 +33,7 @@ repos:
         "--extend-ignore=E203,W503"
     ]
 - repo: https://github.com/pre-commit/mirrors-mypy
-  rev: v0.812
+  rev: v0.991
```

Comment:
`mypy` isn't the issue, this version shouldn't be changed.

The errors are from `black`.

Please revert the mypy version and run

```
pre-commit install
pre-commit run --all
```
it should be clear where the error is after that

Reply:
Long story short, it fixed the `black` issues on my end (I am a Mac user). However, I also raised a lot of other fixes when running the `pre-commit` tests locally. Is there a reason for using this specific version of `mypy`? 

I will surely revert the `mypy` version and try to solve my orginal problem in a different way.